data-changes/location-codes.md

```diff
@@ -1,45 +1,55 @@
+Location ID: **A 2 character code used to uniquely identify different data streams at a single station.** These IDs are commonly used to logically separate multiple instruments or sensor sets at a single station.
 
 ## Background
-Location codes naming conventions used for GeoNet seismic stations are documented [on our _delta_ metadata repository](https://github.com/GeoNet/delta/tree/main/docs).
-Based on this nomenclature convention:
-- all strong motion sensors have a location code `2?`;
-- when a sensor is moved more than 1 m (but less than 200m), its location code will change accordingly (e.g. from `20` to `21`).
+- The Miniseedformat naming international conventions are available at (https://ds.iris.edu/ds/nodes/dmc/data/formats/seed/)  
+- The GeoNet seismic stations Location codes naming conventions are documented on "https://www.geonet.org.nz/data/supplementary/channels".
 
-This convention aligns with [naming requirements for Seismohraph Stations](http://www.isc.ac.uk/registries/registration/) established by the International Seismological Centre (ISC).
+    [More details about GeoNet location codes](https://github.com/GeoNet/delta/blob/main/docs/SEISMIC_SITE_NAMING_CONVENTIONS.md)
 
-For GeoNet seismic sensors, the convention was established in September 2019.
+For GeoNet seismic sensors, a convention was finalized and documented in September 2019.
 
-### Change description
-After the convention was confirmed, we conducted an audit on all equipment changes across the GeoNet National Strong motion sensor network. We found that a number of SM sensors were moved to a new location, without having their location code changed accordingly.
+Based on these nomenclature conventions:
+- **All GeoNet strong motion sensors have a location code `2?`**
+- **When a GeoNet station sensor is moved more than 1 m (but less than 200m), its location code will change accordingly (e.g. from `20` to `21`)**
 
-We are now retroactively correcting our data archive and our metadata records, so that this change is properly represented.
+### Change description
+After the convention was confirmed, we conducted an audit on all equipment changes across the GeoNet National Strong motion sensor network. 
+We found that a number of SM sensors were moved to a new location at the time without having their location code changed accordingly.
 
+We are now (2021) retroactively correcting our data archive and our metadata records, so that this change is properly represented.
 For those SM stations, location codes that were previously recorded as `20` are now being changed to `21`.
 
-Due to the complexity of this work, there has been for a period of time a mismatch between the metadata (that were correctly reporting 21) and the data in the archive. This has now been fixed.
+Due to the complexity of this work, there has been sometimes and  for a limited period of time a mismatch between the metadata (that were correctly reporting 21) and the data in the archive. 
```

Comment:
Just a suggestion: 
replace with: "The complexity of this work is confined in the alignment of archived data with the metadata (e.g. changes in location codes).